src/app/components/Bismillah.tsx

```diff
@@ -0,0 +1,12 @@
+import { Amiri } from "next/font/google";
+
+const arabic_font = Amiri({ weight: "400", subsets: ["arabic"] })
```

Comment:
Why are you trying to use python `snake_case` naming convention in JS. JS uses camelCase for variable naming, let's stick to that.

Reply:
Noted!
Bad habit... sorry for that.